src/v/redpanda/admin_server.cc

```diff
@@ -3181,9 +3179,17 @@ ss::future<ss::json::json_return_type> admin_server::sync_local_state_handler(
         vlog(logger.info, "Need to redirect bucket syncup request");
         throw co_await redirect_to_leader(*request, ntp);
     } else {
-        auto result = co_await _archival_service.map_reduce(
-          manifest_reducer(), [ntp](archival::scheduler_service& service) {
-              return service.maybe_truncate_manifest(ntp);
+        auto result = co_await _partition_manager.map_reduce(
```

Comment:
We should probably depricate this api because we have retention controls now.

Reply:
Yeah -- I think we can deprecate it when doing our other planned improvements to the admin API.

Making the partition reload its manifest state will still be a useful debugging primitive, but with different name, more tests etc.